src/console_reporter.cc

```diff
@@ -44,22 +47,39 @@ bool ConsoleReporter::ReportContext(const Context& context) {
       color_output_ = false;
   }
 #endif
-  std::string str = FormatString("%-*s %13s %13s %10s\n",
-                             static_cast<int>(name_field_width_), "Benchmark",
-                             "Time", "CPU", "Iterations");
-  GetOutputStream() << str << std::string(str.length() - 1, '-') << "\n";
 
   return true;
 }
 
+void ConsoleReporter::PrintHeader(const Run& run) {
+  std::string str = FormatString("%-*s %13s %13s %10s",
+                                 static_cast<int>(name_field_width_),
+                                 "Benchmark", "Time", "CPU", "Iterations");
+  if(!run.counters.empty()) {
+    str += " UserCounters...";
```

Comment:
To be clear, what I wanted to say here is that if you find the use of the ellipsis awkward, say so and I'll remove it.